_commands/tools/nslookup.md

```diff
@@ -2,24 +2,23 @@
 ---
 
 nslookup
--------
-
-nslookup which stands for "name server lookup" is used to query internet name servers interactively for information. 
```

Comment:
Why'd you get rid of the explanation here? I think it's valuable. At least say something about how you can use `nslookup` to translate DNS names to IP addresses and vice versa.